README.md

```diff
@@ -58,60 +48,55 @@ One consequence of centralized script addresses is the necessity for liquidity p
 | Scaling Bottlenecks | Batchers, Execution/Routing Engines, and/or other middlemen | Middlemen can take advantage of their position between users and the protocol. Even if MEV is mitigated, more users --> more execution demand --> centralization of middlemen |
 
-The key takeaway here (especially from the last point) is even 
+Of course, this is not an exhaustive list, and even if some workarounds can be somewhat effective, the underlying design *principles* are suboptimal.
 
-Of course, this is not an exhaustive list, but here is the key takeaway: **even if users control their own keys, for a DEX to achieve true scale the users must also control their own *addresses*.**
+Centralized script addresses necessitate the use of LPs, which imposes a bottleneck on the parallelism of orders. No matter how performant a system of batchers is, this limitation is unavoidable. Additionally, the more decoupled delegation control is from the owner, the more distorted Ouroboros' game theory becomes. It is difficult to predict the extent of this distortion, so minimizing it is of critical importance.
 
+All of this is to say that, much like Bittorrent and the CSL-CCL stack, the best p2p protocols are ones that scale in *proportion* to the number of users. DEXes are no different. With this in mind, here is the key takeaway: **even if users control their own keys, for a DEX to achieve true scale its users must also control their own *addresses*.** 
 
-if some workarounds *can* be effective, the underlying design *principles* are inadequate. Much like Bittorrent, and the CSL-CCL stack, **the best p2p protocols are ones that scale in proportion to the number of participants**. DEXes are no different. With this in mind, we can explore alternative design principles:
+Cardano-Swaps achieves batcher/router-free scalability *with* delegation control via a novel combination of user-controlled script addresses and Beacon Tokens (both are expanded upon below).
 
 ### Programmable Swaps
-Programmable Swaps are a better design choice than liquidity pools. They were first proposed by Axo (formerly known as Maladex). However, in Axo's original [whitepaper](https://www.axo.trade/whitepaper.pdf), there is no mention of giving users *full* delegation control of their own assets. 
+Programmable Swaps are a more promising design choice than liquidity pools. They were first proposed by Axo in their original [whitepaper](https://www.axo.trade/whitepaper.pdf), although there is no mention of giving users *full* delegation control over their assets. In essence, programmable swaps are simply locked UTxOs guarded by a user's own script address. They may be used as inputs, if and only if the resulting TX outputs adhere to the swap logic of the script address. This design pattern scales naturally, since there must be at *least* as many swap addresses as there are users. 
```

Comment:
> In essence, ...

Are you talking about Axo's version or cardano-swaps? I don't think Axo gives each user their own script address.